### PR TITLE
[server][da-vinci-client] Add compaction cancelled metric, add details to node_removable 

### PR DESCRIPTION
**Pull request by majisourav99**
## Add compaction cancelled metric, add details to node_removabl

Added compaction cancellation metric to get more insight on rocksdb SST file size usage. Also adds more details to node removable result to indicate why a node considered removable as passing wrong ID silently tells its removable.

## How was this PR tested?

GHCI
## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: services/venice-controller/src/main/java/com/linkedin/venice/controller/InstanceStatusDecider.java [hunk 1]
```diff
@@ -67,7 +67,8 @@ static NodeRemovableResult isRemovable(
     try {
       // If instance is not alive, it's removable.
       if (!HelixUtils.isLiveInstance(clusterName, instanceId, resources.getHelixManager())) {
-        return NodeRemovableResult.removableResult();
+        return NodeRemovableResult
+            .removableResult("Instance " + instanceId + " not found, either dead or wrong instance ID");
```
Comment (sushantmane) [inline services/venice-controller/src/main/java/com/linkedin/venice/controller/InstanceStatusDecider.java:71]: Or we can just say it's not part of the liveInstance for this cluster anymore?

**Review by sushantmane — DISMISSED** [review]
lgtm. Left a minor comment. Thanks, @majisourav99 !

File: services/venice-controller/src/main/java/com/linkedin/venice/controller/InstanceStatusDecider.java (hunk 1 above)
Comment (majisourav99) [reply services/venice-controller/src/main/java/com/linkedin/venice/controller/InstanceStatusDecider.java]: Sure

**Review by sushantmane — APPROVED** [review]
Thanks!

**Pull request merged**